packages/sdk/src/swap/v2/types.ts

```diff
@@ -86,31 +86,18 @@ interface EgressFields {
 interface SwapStatusResponseCommonFields extends ChainsAndAssets {
   swapId: string;
   destAddress: string;
+  depositChannel: DepositChannelFields | undefined;
   ccmParams: CcmParams | undefined;
   boost: Boost | undefined;
   estimatedDurationSeconds: number | null | undefined;
   srcChainRequiredBlockConfirmations: number | null;
 }
 
-interface VaultSwapCommonFields extends Exclude<SwapStatusResponseCommonFields, 'boost'> {}
```

Comment:
technically vault swaps can't be boost or dca but yeah i'm not sure if we gain much by making it explicit in types

Reply:
true! but i think we should communicate that via the docs rather than the types if it allows us the make the types easier to understand